src/helpers/sentry.ts

```diff
@@ -2,6 +2,10 @@ import * as Sentry from '@sentry/node';
 import type { Express } from 'express';
 
 export function initLogger(app: Express) {
+  if (process.env.NODE_ENV !== 'production') {
```

Comment:
let's add similar lines to `fallbackLogger` and `capture` with respect to their purposes